imports/lib/models/Base.ts

```diff
@@ -53,11 +53,10 @@ class Base<T extends BaseType> extends Mongo.Collection<T> {
     });
   }
 
-  // @ts-ignore Because the Mongo.Collection doesn't know about SimpleSchema
```

Comment:
Huh, did this get smarter?

Reply:
I'm not entirely sure why we ever needed it, since we're already casting to `any` all over the place